contexts/did-v1.jsonld

```diff
@@ -17,6 +17,11 @@
     "SchnorrSecp256k1VerificationKey2019": "sec:SchnorrSecp256k1VerificationKey2019",
     "X25519KeyAgreementKey2019": "sec:X25519KeyAgreementKey2019",
     "ServiceEndpointProxyService": "didv:ServiceEndpointProxyService",
+    "alsoKnownAs": {
+      "@id": "sec:alsoKnownAs",
```

Comment:
```suggestion
      "@id": "as:alsoKnownAs",
```

`alsoKnownAs` is being defined in ActivityStreams, right?

We will also want to either 1) define the `as` prefix -- which we probably shouldn't do, or 2) use full URLs everywhere -- which will bloat the document, but since it's cached, that shouldn't matter.

Reply:
Why shouldn't we define the `as` prefix? I had expected to do that, but ivan beat me to the PR.

Reply:
@rhiaro I think because defining prefixes is now considered some kind of anti pattern.... 

https://github.com/w3c-ccg/security-vocab/issues/57

as is using URLS that don't exist to define vocabulary :)

I'd prefer to see the full stable URL added here.

something like https://www.w3.org/TR/activitystreams-core/#alsoKnownAs

Reply:
I assume it would be defined near here https://www.w3.org/TR/activitystreams-vocabulary/#dfn-id ?

Reply:
The URL would be https://www.w3.org/ns/activitystreams#alsoKnownAs

It would be an extension to ActivityStreams, rather than an addition to core (amending the REC is basically impossible without a WG responsible for it at the moment). Getting it added as an extension will need to go through the Social CG, which is something I'm trying to kick off but don't have an ETA yet.

Reply:
> I assume it would be defined near here https://www.w3.org/TR/activitystreams-vocabulary/#dfn-id ?

No, it's _defined_ in the DID Core spec. That has just been merged into the ED. Then it would be (if all goes to plan) added to the AS2 JSON-LD context and namespace doc which links to the spec definition in DID Core.

Reply:
@iherman @msporny will did core have a similar document? 

https://www.w3.org/TR/activitystreams-core/
https://www.w3.org/ns/activitystreams#alsoKnownAs

https://www.w3.org/TR/did-core/
https://www.w3.org/ns/did/#alsoKnownAs

?

What is the difference between the document we would need for `https://www.w3.org/ns/did/` and the did spec registries?.... seems like a lot of code / term duplication... between did core, did registries and some namespace document....

Reply:
sidenote, I was curious about this, so I setup this site:

https://ns.did.ai/did/#publicKeyJwk

^ if you name a file `index.json`... it gets returned with `content-type` `application/json` for the URL of the directory its in.... which means....

https://ns.did.ai/did/v1 works as expected....

Reply:
As this term is _not_ defined in the Activity Stream vocabulary at present it is not), and, @rhiaro said, it _is_ defined in the DID Core specification with description and all, I do not see why this term would not be added to the did core vocabulary. I have absolutely no idea when and how the AS community will do any change on their own vocabulary, and we cannot be dependent on that.

Reply:
@OR13 wrote:
> What is the difference between the document we would need for https://www.w3.org/ns/did/ and the did spec registries?

No difference... `https://www.w3.org/ns/did` should resolve to the DID Spec Registries document when text/html` is requested. We may want to sprinkle it with RDFa and/or provide an 'application/ld+json' representation of it as well. Again, not mandatory, but given the time, we would do that too.